lib/asciidoctor/converter/html5.rb

```diff
@@ -256,33 +256,35 @@ def convert_document node
 
     if node.attr? 'stem'
       eqnums_val = node.attr 'eqnums', 'none'
-      eqnums_val = 'AMS' if eqnums_val.empty?
-      eqnums_opt = %( equationNumbers: { autoNumber: "#{eqnums_val}" } )
+      eqnums_val = 'ams' if eqnums_val.empty?
```

Comment:
The value is now case-sensitive and strict in MathJax 3. We need to lowercase the value and check if the value is `all`, `none` or `ams`. If the value is invalid, we should use `ams` as a fallback value to be backward compatible with MathJax 2.7.